javascript/entities/test/mock_player.js

```diff
@@ -116,6 +116,24 @@ export class MockPlayer extends Player {
         });
     }
 
+    // ---------------------------------------------------------------------------------------------
```

Comment:
Ehm, we are mocking in here. This should mean no pawn calls! Still you do need to track weapons the player has and such.
Please, also keep track of correct indentation. :)

Reply:
What's wrong with the indentation?

The pawnInvokes are mocked in tests. Resulting in the ability to check what pawnInvvokes are done thus they're here.

Reply:
Interesting stuff by Russell. Never seen it used like this before, hence my note.

Nitpicking: line 129 has 1 space more than necessary.
/me runs and hides